connectivity/manifests/client-egress-to-cidr-1111-deny.yaml

```diff
@@ -0,0 +1,16 @@
+# This policy allows packets towards 1.1.1.1, but not 1.0.0.1.
+# Both addresses are owned by CloudFlare/APNIC.
```

Comment:
Can you explain this a little bit more? From my local testing all traffic is being denied.

Reply:
Also, it looks like this file is not used anywhere.

Reply:
good pick, I supposed to adjust this file, and raise one issue for deny with except policy.

Reply:
here is the github issue https://github.com/cilium/cilium/issues/20914, I have reduced the scope of this test here.

Reply:
Seems like this is expected behavior, I will update docs as suggested in above github issue.